### DISCUSSION TITLE
Merging work and personal account

### DISCUSSION THREAD

**Discussion by dea14** [community/community#22017] [New to GitHub]
<p>Hi, I have two accounts one for my work organization call it <a href="mailto:a@org.com">a@org.com</a> and my personal <a href="mailto:a@me.com">a@me.com</a>, the work account is associated with the work email which will be deactivated after I complete my internship. Is it possible to get my commits from my work account transferred to my personal account?</p>
<p>Thanks!</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>After you delete the work account, you can add the email address to your personal account (even if the address doesn’t work any more). See:</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-user-account-settings/merging-multiple-user-accounts">
  <header class="source">
      <img src="https://docs.github.com/assets/cb-803/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-user-account-settings/merging-multiple-user-accounts" target="_blank" rel="noopener">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/5814919/181068269-d7180b11-7e3f-4987-9873-0146696c0d40.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-user-account-settings/merging-multiple-user-accounts" target="_blank" rel="noopener">Merging multiple user accounts - GitHub Docs</a></h3>

  <p>If you have separate accounts for work and personal use, you can merge the accounts.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
